gm4_liquid_tanks/data/gm4_liquid_tanks/function/apply_output.mcfunction

```diff
@@ -2,8 +2,17 @@
 # @s = processing tank
 
 # merge count from input and set to tank
-data modify storage gm4_liquid_tanks:temp/tank output.count set from storage gm4_liquid_tanks:temp/tank input_slot.count
-data modify block ~ ~ ~ Items[{Slot:0b}] set from storage gm4_liquid_tanks:temp/tank output
+item_modifier gm4_liquid_tanks:apply_output_count:
+    function: "minecraft:set_count"
+    count:
```

Comment:
There's not really any advantage. It's a part of experimenting with the bolt syntax to see what feels most useful for gm4. Since this is the only place this modifier is used, it makes sense to define it here instead of in another file for clarity, and that's what this syntax does in build time. 

Is there any theoretical downsides to vanilla inlining? Like in repeated parsings for example?

Reply:
Performance wise, vanilla inlining should perform just as well as a reference. During load they're converted to equivalent data structures.